table/queries/merged_reduced_scans.sql

```diff
@@ -194,14 +196,15 @@ WITH
     IF(is_control, "CONTROL", domain) as domain,
     category,
     "HTTPS" AS source,
-    country,
+    country_name as country,
     netblock,
     organization,
     controls_failed,
     CleanError(error) AS result,
     ClassifyError(error, "HTTPS", success, blockpage, page_signature) as outcome,
     count(1) AS count
   FROM `firehook-censoredplanet.base.https_scan`
+  JOIN `firehook-censoredplanet.metadata.country_names` ON country_code = country
   GROUP BY date, source, country, category, domain, netblock, organization, controls_failed, result, outcome
 )
 SELECT *
```

Comment:
Do the left join here instead to avoid the duplication.

I think you can do something like:
```sql
SELECT
  AllScans.* except (country),
  IFNULL(country_name, country) as country_name
FROM AllScans
LEFT JOIN `firehook-censoredplanet.metadata.country_names` ON country_code = country
```
Let's rename the field to `country_name` instead of `country` to avoid confusion, especially given `country` means different things in the pipeline. Ideally we would also change the pipeline to use `country_code` instead of `country`, but no need to do it now.

The IFNULL is so that we can see missing mapping.

Reply:
done